.env.sample

```diff
@@ -0,0 +1,2 @@
+REACT_APP_DB_HOST=
+BACKEND_PORT=
```

Comment:
This, on the other end, seems legit. But the variable name *must* begin with `REACT_APP_` to be recognized :shrug: